apps/highlight/src/api/schema.gql

```diff
@@ -15,6 +26,7 @@ type Alias {
 
 type Topic {
   id: String!
+  category_id: Int!
```

Comment:
Can we get the `Category` entity here? We want to show the category name on the topic page